### PR TITLE
Add method to get columns by Entity class.

### PR DESCRIPTION
**Pull request by alexander-aleksandrov**
We need to have a method that can give a collection of columns by entity class in order to use it by reflection in Jdbc module.

### CODE REVIEW

**Comment by alexander-aleksandrov** [comment]
@dmdashenkov PTAL

File: server/src/main/java/io/spine/server/entity/storage/Columns.java [hunk 1]
```diff
@@ -157,6 +157,20 @@ static Column findColumn(Class<? extends Entity> entityClass, String columnName)
...
 
     /**
+     * Retrieves a {@link Collection} of {@link Column Columns} from the given Entity class.
+     *
+     * <p> This method is used in Jdbc table creation process.
+     *
+     * @param entityClass the class containing the {@link Column} definition
+     * @return a {@link Collection} of {@link Column} corresponded to entity class
+     */
+    static Collection<Column> getColumns(Class<? extends Entity> entityClass) {
```
Comment (dmdashenkov) [inline server/src/main/java/io/spine/server/entity/storage/Columns.java]: Please test this method properly.

File: server/src/main/java/io/spine/server/entity/storage/Columns.java [hunk 2]
```diff
@@ -157,6 +157,20 @@ static Column findColumn(Class<? extends Entity> entityClass, String columnName)
...
+        checkNotNull(entityClass);
+        ensureRegistered(entityClass);
+        return knownEntityProperties.get(entityClass);
```
Comment (dmdashenkov) [inline server/src/main/java/io/spine/server/entity/storage/Columns.java]: Please extract a `result` variable for the debugging purposes.

File: server/src/main/java/io/spine/server/entity/storage/Columns.java [hunk 3]
```diff
@@ -157,6 +157,20 @@ static Column findColumn(Class<? extends Entity> entityClass, String columnName)
     }
 
     /**
+     * Retrieves a {@link Collection} of {@link Column Columns} from the given Entity class.
+     *
+     * <p> This method is used in Jdbc table creation process.
```
Comment (dmdashenkov) [inline server/src/main/java/io/spine/server/entity/storage/Columns.java]: This detail is irrelevant.

File: server/src/main/java/io/spine/server/entity/storage/Columns.java (hunk 1 above)
Comment (alexander-aleksandrov) [inline server/src/main/java/io/spine/server/entity/storage/Columns.java]: Done

**Comment by alexander-aleksandrov** [comment]
@dmdashenkov PTAL

**Pull request merged**